plexapi/myplex.py

```diff
@@ -1117,9 +1118,9 @@ def link(self, pin):
 
     def _toOnlineMetadata(self, objs, **kwargs):
         """ Convert a list of media objects to online metadata objects. """
-        # TODO: Add proper support for metadata.provider.plex.tv
+        # TODO: Add proper support for discover.provider.plex.tv
         # Temporary workaround to allow reloading and browsing of online media objects
-        server = PlexServer(self.METADATA, self._token, session=self._session)
+        server = PlexServer(self.DISCOVER, self._token, session=self._session)
```

Comment:
This might be the one exception to leave as `METADATA`.

Reply:
I don't understand why.
All Plex Web app requests are sent to `DISCOVER`.

Reply:
And now that I think about it some more, you can add unrated content to your watchlist and also mark it played/unplayed. I don't know how the Plex UI would handle this, but it works through the API.

So maybe only the `searchDiscover` URL should change, and just leave the rest of them as `metadata`.

Reply:
Ok, back to first commit only.